Assets/Scripts/UI/PickupPup/MainMenu/MainMenu.cs

```diff
@@ -179,11 +184,47 @@ public void OnGiftsClick()
 
     public void OnWatchAdClick()
     {
-        // Disabled
-        EventController.Event(k.GetPlayEvent(k.EMPTY));
+
+        #if UNITY_IOS
+                Advertisement.Initialize(IOS_ID, true);
```

Comment:
From the documentation, it appears the convention is to let ads auto-initialize if not in Editor: https://docs.unity3d.com/ScriptReference/Advertisements.Advertisement.Initialize.html. Is there a reason we have these init calls here?